From b5baf15b36107925186d89af9097b4b8e2d7cad6 Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Wed, 21 Sep 2016 08:38:21 +0200
Subject: [PATCH] MCONFIG.in: handle CLOCK_GETTIME_LIB

Commit 62bc243f1bfe623312a350e4cd0d2e17a69aac45 ("fuse2fs: might need
librt") added a check for the clock_gettime() to verify whether linking
with the librt library is needed or not to use this function.

However, this commit forgot the change in MCONFIG.in that allows to
define the make variable CLOCK_GETTIME_LIB from the CLOCK_GETTIME_LIB
variable substituted by the configure script. Due to this, from a make
perspective $(CLOCK_GETTIME_LIB) is always empty, and we get link
failure on platforms where clock_gettime() is available in librt.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 MCONFIG.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/MCONFIG.in b/MCONFIG.in
index e82963d..9ba9ce2 100644
--- a/MCONFIG.in
+++ b/MCONFIG.in
@@ -125,6 +125,7 @@ LIBSUPPORT = $(LIBINTL) $(LIB)/libsupport@STATIC_LIB_EXT@
 LIBBLKID = @LIBBLKID@ @PRIVATE_LIBS_CMT@ $(LIBUUID)
 LIBINTL = @LIBINTL@
 SYSLIBS = @LIBS@
+CLOCK_GETTIME_LIB = @CLOCK_GETTIME_LIB@
 DEPLIBSS = $(LIB)/libss@LIB_EXT@
 DEPLIBCOM_ERR = $(LIB)/libcom_err@LIB_EXT@
 DEPLIBUUID = @DEPLIBUUID@
-- 
2.7.4

